CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ list(APPEND CMAKE_MODULE_PATH
 set(LOCAL_INIT_DIR "${CMAKE_BINARY_DIR}/local_init")
 
 # Set our options
-option(BUILD_TESTS ON)
+option(BUILD_TESTS "Build the C++ tests" ON)
```

Comment:
this should be copied to the Herbert top-level `CMakeLists.txt`

Reply:
It's in there already https://github.com/pace-neutrons/Herbert/blob/2cf195a4b21030e1ded288bde4572beff5d022e4/CMakeLists.txt#L11

Reply:
👍